docs-v2/integrate/guides/receive-webhooks-from-nango.mdx

```diff
@@ -28,8 +28,7 @@ Webhooks from Nango are POST requests with the following body:
     "model": "<string>",
     "responseResults": { "<DataModel>": { "added": 123, "updated": 123, "deleted": 123 } },
     "syncType": "INITIAL" | "INCREMENTAL",
-    "updatedAfter": "<timestamp>", // Null for initial sync runs.
-    "queryTimeStamp": "<timestamp>", // DEPRECATED: (use updatedAfter) null for initial sync runs.
```

Comment:
We should probably remove it from our code then no?

Reply:
The problem is that it would be a breaking change! Our plan was to 1. deprecate 2. warm about removal + log usage 3. remove entirely.

Reply:
yeah sorry, what I meant is that you removed it from the doc so it felt like all those work had been done